devito/finite_differences/derivative.py

```diff
@@ -192,22 +195,42 @@ def T(self):
             adjoint = direct
 
         return Derivative(self.expr, *self.dims, deriv_order=self.deriv_order,
-                          fd_order=self.fd_order, side=self.side, stagger=self.stagger,
-                          transpose=adjoint)
+                          fd_order=self.fd_order, side=self.side, transpose=adjoint,
+                          x0=self.x0)
+
+    def eval_at(self, var):
```

Comment:
I don't understand in what case the user might want to call `eval_at` explicitly

the way I see it, this is a private method